rest-api/dao/biobank_order_dao.py

```diff
@@ -333,10 +390,150 @@ def to_client_json(self, model):
     resource.collected_info = self._to_handling_info(model.collectedUsername, model.collectedSiteId)
     resource.processed_info = self._to_handling_info(model.processedUsername, model.processedSiteId)
     resource.finalized_info = self._to_handling_info(model.finalizedUsername, model.finalizedSiteId)
+    resource.amendedReason = model.amendedReason
+
+    if model.orderStatus == BiobankOrderStatus.CANCELLED:
+      resource.status = str(BiobankOrderStatus.CANCELLED)
+      resource.cancelledInfo = self._to_handling_info(model.cancelledUsername,
+                                                       model.cancelledSiteId)
+
+    restored = getattr(model, 'restoredSiteId')
+    if restored:
```

Comment:
So what happens if an order gets restored and then canceled again? This logic will set its status to UNSET.

I'm not clear on whether we care about including cancelledInfo / restoredInfo / amendedInfo in the response when the status doesn't match. If we don't, then we can just change this if and the next one to be in else blocks.  If we do, then you probably need to handle resource.status separate from cancelledInfo/restoredInfo/amendedInfo.